src/globemap.ts

```diff
@@ -539,8 +539,8 @@ module powerbi.extensibility.visual {
                         return tileCache;
                     });
             } else {
-                for (let level: number = 1; level <= GlobeMap.maxResolutionLevel; ++level) {
-                    this.mapTextures.push(this.createTexture(level, tileCache[level - 1]));
+                for (let level: number = 2; level <= GlobeMap.maxResolutionLevel + 1; ++level) {
```

Comment:
Please move start value (`2`) into a static variable and use this variable instead of specifying magic numbers